lua/rustaceanvim/executors/vimux.lua

```diff
@@ -2,14 +2,18 @@ local shell = require('rustaceanvim.shell')
 
 ---@type rustaceanvim.Executor
 local M = {
-  execute_command = function(command, args, cwd, _)
+  execute_command = function(command, args, cwd, opts)
+    local envs = ''
```

Comment:
I don't see any other way to implement `env` for this executor; am I missing the `env` option somewhere?